modules/ui.js

```diff
@@ -0,0 +1,36 @@
+import Store from './store.js';
+
+export default class UI {
+  static displayBooks() {
+    // eslint-disable-next-line no-use-before-define
```

Comment:
- [ ] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.